serving/samples/helloworld-scala/helloworld-scala.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: helloworld-scala
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: "your_repository_name/helloworld-scala:0.0.1"
+            imagePullPolicy: IfNotPresent
+            env:
+              - name: MESSAGE
+                value: "Scala & Akka on Knative says hello!"
+              - name: HOST
+                value: "localhost"
+              - name: PORT
+                value: "8080"
+          ports:
```

Comment:
[`RevisionTemplateSpec`](https://github.com/knative/docs/blob/master/reference/serving.md#RevisionTemplateSpec) does not have a `ports` field. Did you mean to put this in `container`? (If so, you want to look at the fields for [`ContainerPort`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#containerport-v1-core), but I think this indicates you don't need to specify `ports` at all.)